### DISCUSSION TITLE
[Bug]: Azure SSO - endless loading

### DISCUSSION THREAD

**Discussion by Schluggi** [danny-avila/LibreChat#1521] [Troubleshooting] [answered]
We configured Azure SSO but each time we want to access https://mydomain.tld/oauth/openid by clicking the social auth login button, everything loads endlessly. No timeout or redirect loop, nothing in the logs, just loading. Everything else works fine. `DOMAIN_CLIENT` and `DOMAIN_SERVER` are set to `https://mydomain.tld`. 

Any idea what could be the issue?

**Comment by danny-avila** [comment]
If you have this deployed on a custom domain, it could be dependent on your domain/nginx configuration.

For your sake, I'm running through the setup instructions again to see if I have issues.

**Reply by Schluggi** [reply]
I checked that. Even inside the container the requests hangs. So nginx should not be the problem.

![grafik](https://github.com/danny-avila/LibreChat/assets/43608073/17d66535-25e2-4d4f-9943-b65fbbb5288f)
(we changed the port to 3080)

**Reply by danny-avila** [reply]
I would like to help more but I can't reproduce it.

![image](https://github.com/danny-avila/LibreChat/assets/110412045/711faca9-4639-4939-a636-bbe5c51dc947)

![image](https://github.com/danny-avila/LibreChat/assets/110412045/250f2ddf-474d-4a9b-8786-2ffa339d697a)

It works for me inside/outside of the container.

Someone else reported this issue on [discord](https://discord.librechat.ai) so I'm inquiring with them

**Reply by Schluggi** [reply, marked as answer]
It happens when redis is not working but defined in any case.
In my case I've not setup redis yet so I set `USE_REDIS=false`. But even without having `REDIS_URI` defined at all this was the issue. 

Removing both (`USE_REDIS` & `REDIS_URI`) solved it.

**Reply by danny-avila** [reply]
Thanks for sharing that! It helped out someone else with the same issue. Redis support is not fully built out and I need to update the terminal message to reflect this.
